HHS/Head-Start-TTADP
# [TTAHUB-974] Remove creator and manager notes automatically from approved ARs

## Description of change
Once a report is approved, manager and creator notes are automatically removed from an AR. All historical approved report notes have been removed via a migration.

## How to test
Leave a report with creator and manager notes. Approve it. Notice that the cleanup occurs. Check approved reports historically for creator notes or approval notes, they have been removed.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-974

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
